clerk/tokens_options.go

```diff
@@ -14,3 +16,10 @@ func WithAuthorizedParty(parties ...string) VerifyTokenOption {
 		o.authorizedParties = authorizedParties
 	}
 }
+
+// WithJWK allows to set the JWK to use for verifying tokens without the need to download or cache any JWKs at runtime
+func WithJWK(jwk *jose.JSONWebKey) VerifyTokenOption {
```

Comment:
It'd be more user-friendly to allow passing the JWT Verification Key directly as a string and then use https://pkg.go.dev/go.step.sm/crypto/jose#ParseKey to turn it into a JWK. Let me know what you think. 🤔

Reply:
Or possibly even supporting both `WithJWK` and `WithJWTVerificationKey`.

Reply:
Yes your instinct is right, i also believe that passing the key as a string and then parsing it, would lead to better UX as the user won't forced to use the package of our choice (jose) in their codebase.

Moving forward, i would suggest to change the signature of the `VerifyTokenOption` to also return an error 
`type VerifyTokenOption func(*verifyTokenOptions) error` so that if the verification key parse fails, we will return early. 

Let me know if you need any help on that

Reply:
> Moving forward, i would suggest to change the signature of the `VerifyTokenOption` to also return an error

I was actually thinking the same when working on the code yesterday 😄

Reply:
Changed it to `WithJWTVerificationKey`, but having trouble with testing. See the other comment.

Reply:
I also changed the signature of `VerifyTokenOption` as suggested. Let me know if that's better done in another PR.